src/javascript/binary/websocket_pages/user/ico_portfolio.js

```diff
@@ -30,9 +30,11 @@ const ICOPortfolio = (() => {
     };
 
     const createPortfolioRow = (data, is_first) => {
-        const long_code  = data.longcode;
-        const ico_status = (State.getResponse('ico_status.ico_status') || '').toLowerCase();
-        // Default to cancel bid. Ended on a button doesn't make sense.
+        const long_code          = data.longcode;
+        const ico_status         = (State.getResponse('ico_status.ico_status') || '').toLowerCase();
+        const deposit_percentage = +((data.shortcode || '').split('_')[2] || State.getResponse('ico_status.initial_deposit_percentage'));
+        const deposit_factor     = (deposit_percentage || 5) / 100;
```

Comment:
is it worth changing to `+deposit_percentage`?

Reply:
@negarn  Doing it right above the line.

Reply:
you're right, thanks :)